MicrosoftDX/nether
# ARM Deployment template for SQL Leaderboard

### Issue: #184 


### Description:
ARM template:
1. Deploys an Azure SQL Server and database
2. Restores the database from a bacpac file, located in an Azure storage, provided by a URI to the file. 

Note - the bacpac file can be found under the deployment/leaderboard-assets folder in this repo

### Test:
1. Copy Leaderboard.bacpac to an Azure container and save the URI
2. Edit the leaderboardSqlDeploy.parameters.json file with your input params (not a must)
3. Deploy the template 
4. Check that Scores tables was created under the database with all required columns:
```sql
SELECT [Id]
      ,[Score]
      ,[GamerTag]
      ,[CustomTag]
      ,[DateAchieved]
  FROM [dbo].[Scores]

```
